editor/editor_node.cpp

```diff
@@ -1001,6 +1001,14 @@ void EditorNode::_resources_reimported(const Vector<String> &p_resources) {
 		}
 	}
 
+	if (scenes.size() > 0) {
```

Comment:
Typically an index check is between >= 0 and < size(), but we check the negated version of that. Size will always be 0 > INT_MAX. So we don't have to check the < 0 case.

Reply:
So based on your comment, I guess the code here is fine ?

Reply:
I think you can simply omit it.